trident.md

```diff
@@ -509,6 +509,15 @@ The specific semantics of the various ways to include or exclude entities are:
 
 If a query results in multiple individuals with the same name, forge will throw an error.
 
+##### Treatment of the genotype data while merging
+
+Forge performs a series of steps to merge the genotype data of multiple source files:
+
+1. Genotype data from each package is streamed in parallel and checks are performed that any SNP location present in any file is outputted (with missing data being filled in for packages where a particular SNP is missing), unless `--intersect` is given, in which case only those position which are present in every package are output.
+2. At each SNP, the consensus alleles are selected, by collecting all reference and alternative alleles from all sources. If more than two non-dummy alleles (alleles different from `N`) are present in that collection, an error is thrown. The two consensus alleles are then chosen and fixed to be reference and alternative allele.
+3. All source genotype data is then read and recoded in terms of the two chosen consensus alleles. This will make sure that source data with flipped reference and alternative allele gets correctly merged in.
+4. SNP ID and genetic position, as part of PLINK `.bim` files is checked across the source files. If there is a mismatch, a debug warning is output (which gets printed to the screen when `--logMode DEBUG` is selected), and simply the first value is chosen to be output into the forged `.bim` file. We decided not to throw an error in such cases, as often the SNP ID and/or the genetic position is simply put to dummy or missing values in some packages but not others, so an error in such cases would be a nuisance.
```

Comment:
Does that mean the SNP identifier is set to some `NA` value in the genotype data?! How do you know which one is which then? I'm aware that a more knowledgeable reader will easily understand this, but for me this sounds very confusing...

Reply:
OK, I can clarify this further, too.